packages/conf/index.d.ts

```diff
@@ -1,5 +1,15 @@
 import type { GeneratorOptions } from "@babel/core";
 
+export type ExtractedMessage = {
+    id: string,
+
+    message?: string,
+    context?: string,
+    origin?: [filename: string, line: number],
+
+    comment?: string,
+}
+
```

Comment:
There is a mess in @lingui/conf typings. They are copy-pasted two times. That's why this type also appears here 2 times